internal/venice-common/src/main/java/com/linkedin/venice/pubsub/manager/TopicManagerStats.java

```diff
@@ -61,6 +69,16 @@ void recordLatency(SENSOR_TYPE sensorType, long startTimeInNs) {
     sensorsByTypes.get(sensorType).record(LatencyUtils.getElapsedTimeFromNSToMS(startTimeInNs));
   }
 
+  // pubSubAdminOpFailureCount
+  void recordPubSubAdminOpFailure() {
+    pubSubAdminOpFailureCount.incrementAndGet();
+  }
+
+  // visible for testing
```

Comment:
The only thing I don't like about `@VisibleForTesting` it adds dependency on guava. Do you think we should have our own annotation for this purpose?

Reply:
Can you check? our code base already has dependencies on guava. We use multiple utils inside that library (e.g., Preconditions). I already have them as part of my GRPC PRs.

Reply:
I know we use Guava, but we've discussed limiting its usage to only gRPC and the modules where gRPC is used.

Reply:
I see them as two separate dependencies and libraries. It is not obvious to me on taking up its dependencies in a subset of module vs overall repo. That said, if you can link to the discussion on why we chose to do that way, it will be helpful to understand context. 

For the sake unblocking this PR, feel free to resolve this comment.

Reply:
I've shared a slack thread with you. Thanks